build/stf-run-ci/tasks/main.yml

```diff
@@ -76,7 +95,33 @@
     tags:
       - deploy
 
-- block:
+- when: __deploy_from_index_enabled | bool
+  tags:
+    - create_bundles
```

Comment:
need to review tag usage now... the logic has changed a fair amount here so some of these tag placements may not make sense any more....

some of this was useful in iterative testing though, and it might be worth adding some notes/docs to README.md on how to use them for certain common scenarios.